doc/user/content/guides/cdc-postgres.md

```diff
@@ -158,15 +158,15 @@ Before deploying a Debezium connector, you need to ensure that the upstream data
 
 1. Grant enough privileges to ensure Debezium can operate in the database. The specific privileges will depend on how much control you want to give to the replication user, so we recommend following the [Debezium documentation](https://debezium.io/documentation/reference/connectors/postgresql.html#postgresql-replication-user-privileges).
 
-1. Set the replica identity value for any tables that you want to replicate but which have **no primary key** defined:
+1. If a table that you want to replicate has a **primary key** defined, you can use your default replica identity value. If a table you want to replicate has **no primary key** defined, you must set the replica identity value to `FULL`:
```

Comment:
We still need to fix this sentence here because even if the table has a primary key you can't use the default replica identity

Reply:
oops, this is debezium. please ignore my comment :)